src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setHasError(true))
+      .finally(() => setIsLoading(false));
```

Comment:
Ensure that the `getPeople` function is implemented correctly to handle API requests and returns a promise that resolves with the expected data structure. This will ensure that the `setPeople` function receives the correct data.